feature-group-definitions/font-synthesis.yml

```
@@ -1,18 +1,7 @@
 name: font-synthesis
-description: The `font-synthesis` CSS property sets whether or not the browser should synthesize bold, italic, small-caps, and other typefaces when they're missing from the font.
-spec: https://drafts.csswg.org/css-fonts-4/#font-synthesis-intro
-usage_stats:
-  - https://chromestatus.com/metrics/css/timeline/popularity/700 # font-synthesis-weight
-  - https://chromestatus.com/metrics/css/timeline/popularity/701 # font-synthesis-style
-  - https://chromestatus.com/metrics/css/timeline/popularity/703 # font-synthesis-small-caps
-  - https://chromestatus.com/metrics/css/timeline/popularity/704 # font-synthesis
+description: The `font-synthesis` CSS shorthand property disables all font synthesis except the given kinds. To disable synthesis of a specific kind of font synthesis, instead use the longhand properties such as `font-synthesis-style` and `font-synthesis-weight`.
```

This is our most opinionated description and I quite like it. 😄

---

I'm glad you like it. I made a typo though:

```suggestion
description: The `font-synthesis` CSS shorthand property disables all font synthesis except the given kinds. To disable a specific kind of font synthesis, instead use the longhand properties such as `font-synthesis-style` and `font-synthesis-weight`.
```